backend/scripts/location_differ.py

```diff
@@ -650,17 +661,12 @@ def location_differ(
                 problem_data["features"].append(prelim_to_problem_json(geojson, msg))
             else:
                 logger.info(
-                    f"{j}/{length}: Found machine to be added: {geojson['properties']['name']}"
+                    f"{j}/{length}: Found machine to be added: {geojson['properties']['name']} in {area}"
                 )
                 changes += 1
                 new += 1
                 machine_idx += 1
                 server_data["features"].append(geojson)
-        if changes > 0:
-            logger.info(
-                f"Location {area} ({i}/{len(areas)}): Changes in {changes}/"
-                f"{len(location_raw_list)} machines found."
-            )
             total_changes += changes
```

Comment:
Minor issue but I think this needs to move one indent to the left (otherwise, `changes` is added multiple times to `total_changes` before it's reset to zero)

Reply:
Great catch!! Thanks :)